contracts/ManagerProxy.sol

```diff
@@ -4,48 +4,42 @@ import "./ManagerProxyTarget.sol";
 
 
 contract ManagerProxy is ManagerProxyTarget {
-    function ManagerProxy(address _controller, bytes32 _targetContractId) Manager(_controller) {
+    function ManagerProxy(address _controller, bytes32 _targetContractId) public Manager(_controller) {
         targetContractId = _targetContractId;
     }
 
     // Based on https://github.com/AugurProject/augur-core/blob/develop/src/libraries/Delegator.sol
     function() public payable {
-        // Set size of method call result to 32 for now until
-        // we can use the RETURNDATACOPY and RETURNDATASIZE opcodes introduced
-        // in the Byzantium hard fork
-        uint32 size = 32;
         address target = controller.getContract(targetContractId);
         // Target contract must be registered
         require(target > 0);
 
         assembly {
             // Load the free memory pointer at 0x40
```

Comment:
Let's consider adding a variable called `freeMemoryPointer` which equals `0x40` initially, since we keep using it and updating the value stored there. Then we could replace all the `0x40`'s throughout the code.